GVRf/Framework/framework/src/main/res/raw/yuv_nv21_to_rgb.fsh

```diff
@@ -0,0 +1,25 @@
+precision highp float;
+in vec2 diffuse_coord;
```

Comment:
layout(location = 0) in vec2 diffuse_coord; though vulkan is going to take it from 0.